subsys/llext/llext.c

```diff
@@ -255,6 +290,12 @@ static int llext_map_sections(struct llext_loader *ldr, struct llext *ext)
 		ldr->sect_map[i] = mem_idx;
 	}
 
+	ldr->prog_data_size = high_shdr.sh_size + high_shdr.sh_offset - low_shdr.sh_offset;
```

Comment:
where is this used?

Reply:
here: https://github.com/thesofproject/sof/blob/749890f7d5d469b82addf55e463534a69df82f93/src/library_manager/llext_manager.c#L204 (sorry, cannot link to a line or even a hunk in a commit, but it's from the SOF side ounterpart https://github.com/thesofproject/sof/pull/8990)